internal/ocm/ocm.go

```diff
@@ -134,13 +136,28 @@ func GetOCMComponentsWithVersions(repo ocm.Repository, components []string, pref
 			Versions: make([]v1beta1.ComponentVersion, 0),
 		}
 
+	outer:
 		for _, version := range versions {
```

Comment:
Can you elaborate why we are jumping out of the for-loop back to the beginning of the for-loop?

Reply:
Thanks for your explanation👍